Networking/Networking/CookieNonce/CookieNonceAuthenticator.swift

```diff
@@ -27,18 +27,17 @@ final class CookieNonceAuthenticator: RequestRetrier & RequestAdapter {
     }
 
     // MARK: Request Adapter
-
-    func adapt(_ urlRequest: URLRequest) throws -> URLRequest {
+    func adapt(_ urlRequest: URLRequest, for session: Alamofire.Session, completion: @escaping (Result<URLRequest, Swift.Error>) -> Void) {
         guard let nonce else {
-            return urlRequest
+            return completion(.success(urlRequest))
```

Comment:
@itsmeichigo This seems fine for the purposes of the PR, but do you think it should have an error result here?

I ask because I don't know the full details of the site credentials login.

Reply:
Hey there - this implementation means that if there is no nonce found, the URL request is sent as is, which will then causing the API to returns 401 error code. We handle the error in `retry(_:for:dueTo:completion)` method below, including store creation authentication to get the nonce. So Tony's change is correct here.